modules/optimization/include/postprocessors/VariableFunctionSideIntegral.h

```diff
@@ -0,0 +1,31 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "FunctionSideIntegral.h"
+
+class Function;
+
+/**
+ * This postprocessor computes the inner product of a function and variable over a sideset
```

Comment:
same comment

Reply:
I'm fine with removing these. I would need some help implementing or using the objects you are talking about.